basics/how-storage-works/filecoin-plus.md

```diff
@@ -75,23 +69,28 @@ The steps a client should follow to acquire DataCap are as follows:
 4. Submit an application for DataCap from an allocator. You can submit a request to FIDL via their [Github application form](https://github.com/fidlabs/Open-Data-Pathway/issues/new/choose) or [Google Form](https://www.fidl.tech/apply-for-datacap).
 5. Use the DataCap in a storage deal.
 
+### Steps to Acquire Testnet DataCap as a Builder
 For builders on the [Calibration testnet](../../networks/calibration/) who need testnet DataCap to test their applications, a faucet is available. The steps a builder should follow to acquire testnet DataCap are as follows:
 
 1. Create a wallet on Filecoin Calibration testnet. For more information, see the [Calibration docs](../../networks/calibration/) or [Github](https://github.com/filecoin-project/testnet-calibration).
 2. Grant the wallet address DataCap by using this [faucet](https://faucet.calibnet.chainsafe-fil.io/datacap.html).
 
-## **Smart contracts**
+## **DataCap for Smart contracts**
 
 Smart contracts can acquire and use DataCap just like any regular client. To do so, simply enter the `f410` address of the smart contract as the client address when making a request for DataCap.
 
-\[!Important] It’s important to note that DataCap allocations are a one-time credit for a Filecoin address and cannot be transferred between smart contracts. If you need to redeploy the smart contract, you must request additional DataCap.
-
-## Spend DataCap
+### Important 
+It’s important to note that DataCap allocations are a one-time credit for a Filecoin address and cannot be transferred between smart contracts. If you need to redeploy the smart contract, you must request additional DataCap.
 
+## How to Use DataCap
 Once you have an address with DataCap, you can make deals using DataCap as a part of the payment. Because storage providers receive a deal quality multiplier for taking Filecoin+ deals, many storage providers offer special pricing and services to attract clients who use DataCap to make deals.
 
+[Learn more about Storage Deals.](https://docs.filecoin.io/storage-providers/filecoin-deals/storage-deals) 
+
 By default, when you make a deal with an address with DataCap allocated, you will spend that DataCap when making the deal.
 
+### How to Use DataCap, using the API 
+
 If making deals through the [API](https://github.com/filecoin-project/lotus/blob/master/documentation/en/api-v0-methods.md#ClientStartDeal), make sure that the `VerifiedDeal` parameter is set to `true` when calling `ClientStartDeal`.
 
```

Comment:
the `Client` api endpoints have all been removed from the lotus daemon, so this section should be removed